src/tests.py

```diff
@@ -236,13 +238,127 @@ def _integration_test(target_directory):
             assert str(e) == "Branch does not have a parent: master"
 
 
+
+def _test_clean_branches(target_directory):
+    # type: (Text) -> None
+    target_directory = os.path.expanduser(target_directory)
+    target_container = os.path.dirname(target_directory)
+    assert not os.path.exists(target_directory)
+    assert os.path.isdir(target_container)
```

Comment:
Would you mind putting this in `run_test` since we're now repeating it 3 times?

Reply:
👍